src/desktop/apps/artwork/components/partner_artworks/index.jade

```diff
@@ -16,6 +17,6 @@ if artwork.partner && artwork.partner.artworks && artwork.partner.artworks.lengt
       include ../../../../components/artwork_masonry/index
 
     .artwork-section__jump
-      a.avant-garde-jump-link( href='#{artwork.partner.href}/works' )
+      a.avant-garde-jump-link( href='#{link}' )
```

Comment:
Does `partner_artworks` have an associated test?

Reply:
Not that I see one and I believe we have to rewrite it for RAP so I went without test here. Though it's a bug and we should sure have one for the new page.

Reply:
👍